<ui version="4.0" >
 <class>cs8WagoNewAddressWidget</class>
 <widget class="QDialog" name="cs8WagoNewAddressWidget" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Wago Coupler Network Configuration</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Enter below the &lt;b>current&lt;/b> IP address of the Wago coupler that you want to reconfigure. Usually the IP address is 192.168.0.1</string>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>&amp;Wago coupler current IP Address</string>
         </property>
         <property name="buddy" >
          <cstring>leAddress</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leAddress" >
         <property name="inputMask" >
          <string>009.009.009.009;_</string>
         </property>
         <property name="text" >
          <string>192.168.000.001</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Enter below the &lt;span style=" font-weight:600;">new&lt;/span> network configuration that you want to configure for this coupler. &lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">W&amp;amp;ago Node new IP Address &lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Default is &lt;span style=" font-weight:600;">192.168.0.2&lt;/span> for the &lt;span style=" font-weight:600;">system&lt;/span> coupler and &lt;span style=" font-weight:600;">192.168.0.3&lt;/span> for the &lt;span style=" font-weight:600;">user&lt;/span> coupler&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
       <property name="buddy" >
        <cstring>leAddress</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="leNewAddress" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="inputMask" >
        <string>009.009.009.009;_</string>
       </property>
       <property name="text" >
        <string>192.168.000.002</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Subnet Mask 
Default is 255.255.255.0</string>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
       <property name="buddy" >
        <cstring>leAddress</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="leNewSubnet" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="inputMask" >
        <string>009.009.009.009;_</string>
       </property>
       <property name="text" >
        <string>255.255.255.000</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Gateway
Default is 0.0.0.0</string>
       </property>
       <property name="buddy" >
        <cstring>leAddress</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="leNewGateway" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="inputMask" >
        <string>009.009.009.009;_</string>
       </property>
       <property name="text" >
        <string>000.000.000.000</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leNewAddress</tabstop>
  <tabstop>leNewSubnet</tabstop>
  <tabstop>leNewGateway</tabstop>
  <tabstop>leAddress</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
